packages/cli/src/commands/new/index.ts

```diff
@@ -9,15 +9,11 @@ export function New(context: IContext): CommandModule<{}, {}> {
   return {
     command: 'new',
     aliases: 'n',
-    describe: 'create a new app project',
+    describe: 'create a new app project. You can set the default language with the "set-lang" command.',
```

Comment:
Will quickstarts need to change b/c of this?

Reply:
looks like Alex has updated it in his docs